docs/deployment/controller.md

```diff
@@ -0,0 +1,56 @@
+# Depoloy dlrover job controller on Minikube cluster
+
+Here we would like to introuce how to depoly dlrover job controller on minikube cluster step by step.
+
+## 1.Preliminary
+- Make sure you have [minikube](https://kubernetes.io/docs/tasks/tools/)  installed and run ``minikube start``
+
+## 2. Create Namespace
+
+```bash
+$ kubectl create namespace dlrover
+```
+
+## 3. Deploy Dlrover Job Controller With Kubectl
+
+```bash
+$ kubectl apply -k /dlrover/go/config/manifests/bases
```

Comment:
this can be a github address like github.com/xx/xx/bases

Reply:
Good idea! An absolute address helps a lot!